resources/constants.rb

```diff
@@ -37,10 +37,6 @@ def self.small
     return 1e-9
   end
 
-  def self.AssumedEvapCoolerEffectiveness
-    return 0.72
```

Comment:
Hard-coded this in hvac.rb, and then updated hvac_sizing.rb to reference the object property rather than having both files reference the constant. That way they are better guaranteed to stay in sync.

Reply:
Oh I did this again, thanks for improving it.